extensions/amp-story/1.0/_locales/en.json

```diff
@@ -218,5 +218,9 @@
   "101": {
     "string": "Shop Now",
     "description": "Label for the shopping CTA button."
+  },
+  "102": {
+    "string": "Also in this story",
```

Comment:
Just checking that this is the title we want to use for the PLP instead of "Featured in this story" or something else? I think the word "also" comes off as a bit awkward since it suggests to me that the PLP will be listing products that specifically are not already visible in the story page.

Reply:
Agree. I pinged design to double check.